src/Legacy/Storage.php

```diff
@@ -694,6 +694,9 @@ function ($word) {
      */
     private function searchSingleContentType($query, $contenttype, $fields, array $filter = null, $implode = false)
     {
+        // Add status to fields
+        $fields['status'] = true;
+
```

Comment:
Something … finger …

Back up in `searchContent()` the [passed in fields](https://github.com/graham73may/bolt/blob/96f24521963422b43623495dc98c114c8c9ff076/src/Legacy/Storage.php#L873-L875) are ConentType field arrays.

The `foreach ($fields as $field => $fieldconfig) {` loop then wants `$fieldconfig['type']` and `$fieldconfig` is a boolean.